packages/terra-clinical-item-collection/src/_DetermineElements.js

```diff
@@ -1,15 +0,0 @@
-const determineElements = (row) => {
```

Comment:
This was utilized by `_TableHeaderCell` which was removed for a different column width implementation.